canonical-web-and-design/ubuntu.com
# Fix engage thank-you pages and <space>&nbsp;

## Done

- Fix engage thank-you pages and `<space>&nbsp;`

## QA

- Check out this feature branch
- Run the site using the command `./run serve` or `dotrun`
- View the site locally in your web browser at: http://0.0.0.0:8001/engage/es/kubernetes-despliegue-empresa-manual/thank-you and http://0.0.0.0:8001/engage/pt/vmware-para-o-openstack/thank-you
- Run through the following [QA steps](https://canonical-web-and-design.github.io/practices/workflow/qa-steps.html)
- See that the a accute (á) works and 'Whitepaper' are capitalised


## Issue / Card

Fixes #8659

## Screenshots

see a acute
![image](https://user-images.githubusercontent.com/441217/98223667-5d66ef00-1f4a-11eb-8038-62dab7cd5b2c.png)

see capitalised whitepaper
![image](https://user-images.githubusercontent.com/441217/98223716-6a83de00-1f4a-11eb-8d53-2340fca3384f.png)
